nxdialog/nxdialog

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python
```

Comment:
@uli42 See [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#web_servers_databases_dynamic_languages_2).

```
In RHEL 8, executable Python scripts are expected to use hashbangs (shebangs) specifying explicitly at least the major Python version.
```

This should be **python2** or even better **python3**

Reply:
i have changed it to python2. I am wondering why I haven't done that right at the beginning.

Reply:
python3 conversion is a bit of a problem since we also want to use it on RHEL6 and RHEL7 where python3 is not available by default.